src/commands/UnbanBanCommand.ts

```diff
@@ -72,6 +72,22 @@ export async function parseArguments(
             }
         }
 
+        if (arg.startsWith("https://matrix.to/#/")) {
```

Comment:
(we can also just skip this `if` statement depending on how comfortable we are with `try/catch` control flow - either is fine)